docs/source/listings.rst

```diff
@@ -197,13 +207,17 @@ BigQuery Tables
       - Table
       - Description
 
-    * - lsst
+    * - lsst_alerts
       - alerts_v7_4
       - Alert data for LSST schema version 7.4. This table is an archive of the lsst-alerts Pub/Sub stream,
         excluding image cutouts and metadata.
         It has the same schema as the original alert bytes (except cutouts), including nested and repeated fields.
         Equivalent tables exist for previous schema versions: alerts_v7_3,  alerts_v7_1.
 
+    * - lsst_value_added
+      - upsilon
+      - Results from an UPSILoN (Kim \& Bailer-Jones, 2015) periodic variable star classification (multi-class).
```

Comment:
Ya, I'm now thinking we should link to the upsilon docs or code repo. That is more likely to have some specific info that is more directly applicable than what's in the paper. Plus, it lists the paper reference (assuming you're using https://github.com/dwkim78/upsilon), so folks will still be able to find that.

Also, if our module is using something extra (like a trained model that is not included with the library) that seems worth mentioning, you could mention that.